api/tests/test_application_status_api.py

```diff
@@ -148,22 +149,30 @@ def test_service_sector_manager_can_create_status(self):
             ApplicationStatus.REVIEW_DONE
         )
 
-
-@pytest.mark.django_db
-class ApplicationEventStatusApiPermissionsTestCase(ApplicationStatusBaseTestCase):
-    def test_service_sector_manager_cant_modify_status(self):
-        response = self.manager_api_client.post(
-            reverse(
-                "application_event_status-detail",
-                kwargs={"pk": self.application_event.id},
-            ),
-            data={
-                "status": ApplicationEventStatus.VALIDATED,
-                "application_event_id": self.application_event.id,
+    @mock.patch("applications.signals.create_aggregate_data_for_application_round")
```

Comment:
actually that is some testing that i did and its partially solved in try except in AggregateDataCreator. I'll remove that mock since it wont do anything

Reply:
I see 😄